src/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -266,42 +266,98 @@ IsEnabled(
     return ( dwFlag & DEBUG_FLAGS_VAR );
 }
 
+void WriteFileEncoded(UINT codePage, HANDLE hFile, const LPCWSTR  szString)
+{
+    DWORD nBytesWritten = 0;
+    auto const encodedByteCount = WideCharToMultiByte(codePage, 0, szString, -1, nullptr, 0, nullptr, nullptr);
+    auto encodedBytes = std::shared_ptr<CHAR[]>(new CHAR[encodedByteCount]);
+    WideCharToMultiByte(codePage, 0, szString, -1, encodedBytes.get(), encodedByteCount, nullptr, nullptr);
+
+    WriteFile(hFile, encodedBytes.get(), encodedByteCount - 1, &nBytesWritten, nullptr);
+}
+
 VOID
-DebugPrint(
+DebugPrintW(
     DWORD   dwFlag,
-    const LPCSTR  szString
+    const LPCWSTR  szString
     )
 {
-    STACK_STRA (strOutput, 256);
+    STACK_STRU (strOutput, 256);
```

Comment:
Were these restored functions that were present before?

Reply:
Nope, before they went the other direction wide strings were narrowed before being logged.